<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deledit</class>
 <widget class="QMainWindow" name="deledit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="listpage_2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="detailedpage_2">
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>74</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Name Of Hotel:</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="adressedit_2">
        <property name="geometry">
         <rect>
          <x>89</x>
          <y>61</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="phonenumberedit_2">
        <property name="geometry">
         <rect>
          <x>89</x>
          <y>285</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>285</y>
          <width>74</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Phone Number:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nameedit_2">
        <property name="geometry">
         <rect>
          <x>89</x>
          <y>9</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QDialogButtonBox" name="buttonBox_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>490</y>
          <width>221</width>
          <height>23</height>
         </rect>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Discard|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_2">
        <property name="geometry">
         <rect>
          <x>89</x>
          <y>35</y>
          <width>83</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>35</y>
          <width>70</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Check in Date:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>61</y>
          <width>37</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Adress:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>407</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="actionEdit"/>
   </widget>
   <addaction name="menuSelect"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
